OsmAnd/src/net/osmand/plus/AppVersionUpgradeOnInit.java

```diff
@@ -175,15 +175,14 @@ public void resetFirstTimeRun(SharedPreferences startPrefs) {
 	}
 
 	private void addCustomProfile() {
```

Comment:
This is incorrect approach cause profiles will be added once during upgrade and they should not be deletable actually

Reply:
Okay, I assumed that they should behave as other custom profiles which user can add with the ability to delete them.